.devcontainer/devcontainer.json

```
@@ -4,7 +4,21 @@
     "dockerfile": "Dockerfile",
     "context": ".."
   },
-  "appPort": "8124:8123",
+  "forwardPorts": [8123, 8100, 8090, 8080],
+  "portsAttributes": {
+    "8123": {
```

```suggestion
  "forwardPorts": [
    "8124:8123",
    8100,
    8090,
    8080
  ],
  "portsAttributes": {
    "8124:8123": {
```
Lets keep the current mapping to not conflict with the core devcontainer.

---

Not needed. The way I had set it will tell vscode to use 8123 if possible, otherwise the next available port.

---

I tested it, it's definitely needed.

---

@ludeeus that's most likely because your are not using https://github.com/home-assistant/core/pull/82753.

---

No, I'm using dev.

---

Yes, so let's try to merge both. I'll fix the needed stuff in https://github.com/home-assistant/core/pull/82753.